.github/workflows/publish-release.yml

```diff
@@ -71,6 +71,17 @@ jobs:
         echo "::set-output name=prerelease::$prerelease"
       shell: bash
 
+    - name: Set up S3cmd and configure AWS credentials
+      uses: s3-actions/s3cmd@v1.1
```

Comment:
I suggest we opt for the official action by Amazon: https://github.com/aws-actions/configure-aws-credentials